src/main/java/bopes/task/Deadline.java

```diff
@@ -0,0 +1,60 @@
+package bopes.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import bopes.exception.BopesException;
+
+/**
+ * The Deadline class represents a task with a specific deadline.
+ * It extends the Task class and includes a LocalDateTime field to store the deadline date and time.
+ */
+public class Deadline extends Task {
```

Comment:
Should this be DeadlineTask instead, since Deadline is not really a task if interpreted without context